client/package.json

```diff
@@ -15,6 +15,7 @@
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.1",
     "@testing-library/user-event": "^12.2.2",
+    "json-server": "^0.16.3",
```

Comment:
Maybe start the json-server within your server.js as described here: https://www.npmjs.com/package/json-server#module